specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-06-04/examples/GetGalleryApplication.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "api-version": "2019-06-04",
+    "Metadata": "true",
+    "incarnationId": "b845c18d-b249-4c7a-a4c3-ffdd51757ccd"
+  },
+  "responses": {
+        "200": {
+            "body": "{application binary package}"
+        },
+        "204": {
+            "body": ""
```

Comment:
This should be removed instead of adding the `"schema": {}`

Reply:
I hadn't thought of that! That's much cleaner; I'll make the change.

Reply:
No worries, I had just been cleaning those up on some other Swaggers files since it didn't used to get caught by OAV